modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/SharePodcastHelper.kt

```diff
@@ -39,17 +48,25 @@ data class SharePodcastHelper(
         } ?: run {
             url = "$host/podcast/${podcast.uuid}"
         }
-        sendText(url)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
I think it would be good to explain why we're doing this version check. I don't feel strongly about the specifics, but something along these lines could work. Alternatively, we could just extract the version check out into a variable named something like `isCapableOfRichContentPreviews`.
```suggestion
        
        // Android 10+ supports rich content previews: https://developer.android.com/training/sharing/send#adding-rich-content-previews
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Reply:
This works for me. Let me know If I can commit this suggestion.

Reply:
Committing that is fine with me. 👍